dolphinscheduler-ui/src/views/projects/task/components/node/format-data.ts

```diff
@@ -528,6 +540,42 @@ export function formatModel(data: ITaskData) {
     params.zeppelinParagraphId = data.taskParams.zeppelinParagraphId
   }
 
+  if (data.taskParams?.condaEnvName) {
+    params.condaEnvName = data.taskParams.condaEnvName
+  }
+
+  if (data.taskParams?.inputNotePath) {
+    params.inputNotePath = data.taskParams.inputNotePath
+  }
+
+  if (data.taskParams?.outputNotePath) {
+    params.outputNotePath = data.taskParams.outputNotePath
+  }
+
+  if (data.taskParams?.parameters) {
+    params.parameters = data.taskParams.parameters
+  }
+
+  if (data.taskParams?.kernel) {
+    params.kernel = data.taskParams.kernel
+  }
+
+  if (data.taskParams?.engine) {
+    params.engine = data.taskParams.engine
+  }
+
+  if (data.taskParams?.executionTimeout) {
+    params.executionTimeout = data.taskParams.executionTimeout
+  }
+
+  if (data.taskParams?.startTimeout) {
+    params.startTimeout = data.taskParams.startTimeout
+  }
+
+  if (data.taskParams?.others) {
+    params.others = data.taskParams.others
+  }
+
```

Comment:
If this is not a nested structure of the taskParams, there is no need to set the value one by one.

Reply:
Thanks for pointing it out. I'm not familiar with the front-end, could you please give some suggestions on how to fix it?

Reply:
I‘ve fixed it, please check.

Reply:
> I‘ve fixed it, please check.

Thx for the help : )